src/lib/tuya.ts

```diff
@@ -1187,6 +1187,61 @@ const tuyaTz = {
             return {state: {do_not_disturb: value}};
         },
     } satisfies Tz.Converter,
+    on_off_countdown: {
+        // Note: This is the Tuya on-off countdown feature documented for switches and smart plugs
+        //       using the Zigbee 'onWithTimedOff' command in a non-standard way.
```

Comment:
What exactly is non-standard here?

Reply:
The standard behavior is described in the Zigbee standard. See sections **3.8.2.3.6 On With Timed Off Command** and **3.8.2.4 State Description ** in the [ZCL](https://zigbeealliance.org/wp-content/uploads/2019/12/07-5123-06-zigbee-cluster-library-specification.pdf). To summarize,  the standard `onWithTimedOff` behavior is `TURN ON, WAIT on_time/10 SECONDS AND THEN TURN OFF`.  The register `off_wait_time` can be used to add an additional delay during which the device cannot be turned on. This is why the current implementation of `on_off` in `converters/Zigbee.ts` only allows 'on_time' to be set with 'state="on"'. 

The Tuya behavior is a lot more generic : "WAIT on_time SECONDS THEN TOGGLE" so a delayed toggle. There is no restriction on the initial state and `on_time` is now given in seconds.

A device that uses the default 'on_off' converter can be tested by turning it off and then `/set { state:"on", "on_time": 6 }`. Because of the scaling by 10 in 'on_off' converter, the value 60 is written in the register `on_time`.

- If the device turns ON immediately and OFF after 6 seconds then the device is using the standard Zigbee behavior described in the ZCL  (probably because I only have Tuya devices)
- If the device stays OFF and turns ON after 60 seconds then the device is using the Tuya countdown behavior.